extensions/search-notion/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "search-notion",
+  "title": "Search Notion Without Admin Access",
+  "description": "Search Notion pages and open visited ones faster.",
+  "icon": "command-icon.png",
+  "author": "reckoning-dev",
+  "version": "1.0.0",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "commandsearchnotion",
```

Comment:
Please fix the name of the file to have consistent case that actually file name.
It's important for Ubuntu system ( as it's case-sensitive) and we used in our CI system

Reply:
yep, that fixed it. Thanks!